resources/js/src/app/services/AutoFocusService.js

```diff
@@ -0,0 +1,47 @@
+import ModalService from "services/ModalService";
+
+export function autoFocus()
+{
+    const modals = document.getElementsByClassName("modal");
+
+    for (const modal in modals)
+	{
+        if (typeof modals[modal] === "object")
+        {
+            const currentModal = ModalService.findModal(modals[modal]);
+
+            if (currentModal)
+            {
+                currentModal.on("shown.bs.modal", () =>
+                {
+                    triggerAutoFocus(currentModal);
+                });
+            }
+        }
+    }
+
+    triggerAutoFocus();
+}
+
+export function triggerAutoFocus(modal)
+{
+    modal = modal || null;
```

Comment:
Wofür ist diese Zeile? Modal wird auch als null gesetzt wenn nichts reingegeben wird in die Funktion. Und falls undefined hier irgendwie reinkommen sollte, wird das durch die if-Abfrage in Zeile 32 abgefangen